conneg-by-ap/index.html

```diff
@@ -498,8 +498,8 @@ <h3>Profile Identification</h3>
       </p>
       <p>
         If a URI is used for profile identification, it is recommended that it be an HTTP
```

Comment:
Should "recommended" be "RECOMMENDED" since the the "MAY" in the next line is also RFC 2119 language?

Reply:
I have changed this to SHOULD so the comment is redundant.